config/initializers/active_force.rb

```diff
@@ -0,0 +1,27 @@
+module ActiveForce
+
+  class << self
+
+    # Use a lazy setting of the client so that migrations etc are in place
+    # to allow the RealClient to be successfully instantiated.
+    alias_method :original_sfdc_client, :sfdc_client
+    def sfdc_client
+      if !original_sfdc_client.is_a?(Salesforce::Client)
+        self.sfdc_client = Salesforce::Client.new
```

Comment:
I haven't looked at Accounts.

What we should do really depends on the goal. Maybe worry later.